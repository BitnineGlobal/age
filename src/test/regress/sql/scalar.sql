--
-- Regression tests for scalar functions
--

-- Test for the tointegerornull() function

RETURN tointegerornull('42'), tointegerornull('not a number'), tointegerornull(true), tointegerornull(['A', 'B', 'C']);

RETURN tointegerornull(12.5), tointegerornull('13.5'), tointegerornull(45), tointegerornull(false);
