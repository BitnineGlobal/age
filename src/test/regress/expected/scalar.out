--
-- Regression tests for scalar functions
--
-- Test for the tointegerornull() function
RETURN tointegerornull('42'), tointegerornull('not a number'), tointegerornull(true), tointegerornull(['A', 'B', 'C']);
 tointegerornull | tointegerornull | tointegerornull | tointegerornull 
-----------------+-----------------+-----------------+-----------------
 42              |                 | 1               | 
(1 row)

RETURN tointegerornull(12.5), tointegerornull('13.5'), tointegerornull(45), tointegerornull(false);
 tointegerornull | tointegerornull | tointegerornull | tointegerornull 
-----------------+-----------------+-----------------+-----------------
 12              | 13              | 45              | 0
(1 row)

